src/utilities/grid.ts

```diff
@@ -60,7 +60,7 @@ export const constraintsGrid: Grid = {
 export const schedulingGrid: Grid = {
   columnSizes: '1fr 3px 3fr 3px 1fr',
   columns: [
-    { componentName: 'SchedulingPanel', id: 1, type: 'component' },
+    { componentName: 'SchedulingGoalsPanel', id: 1, type: 'component' },
```

Comment:
It might be nice to add the split panel here and show the `SchedulingGoalsPanel` and `SchedulingConditionsPanel`